src/shogun/regression/LinearRidgeRegression.cpp

```diff
@@ -39,12 +39,13 @@ void LinearRidgeRegression::init()
 
 	SG_ADD(&m_tau, "tau", "Regularization parameter", ParameterProperties::HYPER);
 	SG_ADD(
-	    &m_use_bias, "use_bias", "Whether or not to fit an offset term");
+	    &m_use_bias, "use_bias", "Whether or not to fit an offset term", 
+	    ParameterProperties::SETTING);
```

Comment:
@karlnapf there is this issue now with crossvalidation that if we don't set a parameter to either hyperparameter or setting it won't be copied... Need to continue going through all the parameters in all algos and set something..

Reply:
yes, I think this whitelisting idea is maybe not the best. What about blacklisting parameters instead in xval? Like readonly ones?

Reply:
Yup, that is probably better... Just need to label all the labels/features as readonly

Reply:
ah no! Thats why we did it this way. Features should not be readonly.
We need to whitelist as otherwise we have no way to distinguish between model parameters and things like feautres. I remember now.
I put a PR in to test exactly such things (explicit xval by hand against the class we have), that they have the same results. If we combine this with random parameter content, this will catch missing parameter registrations